shardingsphere-infra/shardingsphere-infra-binder/src/test/java/org/apache/shardingsphere/infra/binder/segment/select/projection/engine/ProjectionsContextEngineTest.java

```diff
@@ -391,4 +391,49 @@ private void assertCreateProjectionsContextWithTemporaryTable(final SelectStatem
                 .createProjectionsContext(selectStatementContext.getFromSimpleTableSegments(), projectionsSegment, groupByContext, new OrderByContext(Collections.emptyList(), false));
         assertNotNull(actual);
     }
+
+    @Test
+    public void assertCreateProjectionsContextWhenTableNameOrAliasIgnoreCaseForMySQL() {
+        assertCreateProjectionsContextWhenTableNameOrAliasIgnoreCase(new MySQLSelectStatement(), new MySQLSelectStatement());
+    }
+
```

Comment:
These indents are not consistent with the previous one.

Reply:
my idea, it's not flexible